query/src/main/scala/filodb/query/exec/PromQlExec.scala

```diff
@@ -0,0 +1,117 @@
+package filodb.query.exec
+
+import com.softwaremill.sttp.akkahttp.AkkaHttpBackend
+import com.softwaremill.sttp.circe._
+import com.typesafe.scalalogging.StrictLogging
+import monix.eval.Task
+import monix.execution.Scheduler
+import monix.reactive.Observable
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+import filodb.core.DatasetRef
+import filodb.core.metadata.Column.ColumnType
+import filodb.core.metadata.Dataset
+import filodb.core.query._
+import filodb.core.store.ChunkSource
+import filodb.memory.format.RowReader
+import filodb.memory.format.ZeroCopyUTF8String._
+import filodb.query._
+
+case class PromQlExec(id: String,
+                      dispatcher: PlanDispatcher,
+                      dataset: DatasetRef, params: PromQlQueryParams,
+                      submitTime: Long = System.currentTimeMillis()) extends LeafExecPlan {
+
+  protected def args: String = ""
```

Comment:
Include PromQlQueryParams here. It should get printed when the exec plan is printed.